result/rsvplsp.yaml

```diff
@@ -0,0 +1,186 @@
+          x-field-uid: 1
+        destination_address:
+          description: >-
+            The IPV4 destination address of RSVP session.
+          type: string
+          format: ipv4
+          x-field-uid: 2
+        lsp:
+          description: |-
+            It refers to the RSVP LSP properties.
+          $ref: '#/components/schemas/RsvpLsp.State'
+          x-field-uid: 3
+        rros:
+          description: |-
+            It refers to RSVP RRO objects container.
+          items:
+            $ref: '#/components/schemas/RsvpLsp.Ipv4Rro'
+          type: array
+          x-field-uid: 4
+        eros:
+          description: |-
+            It refers to RSVP ERO objects container.
+          items:
+            $ref: '#/components/schemas/RsvpLsp.Ipv4Ero'
+          type: array
+          x-field-uid: 5
+
+    RsvpLsp.State:
+      description: >-
+        IPv4 RSVP-TE Discovered LSPs.
+      type: object
+      properties:
+        tunnel_id:
+          description: >-
+            The tunnel id of RSVP session which acts as an identifier that remains constant over the life of the tunnel.
+          type: integer
+          x-field-uid: 1
+        lsp_id:
+          description: >-
+            The lsp-id of RSVP session which acts as a differentiator for two lsps originating from the same headend,
+            commonly used to distinguish RSVP sessions during make before break operations.
+          type: integer
+          x-field-uid: 2
+        session_name:
+          description: >-
+            The name of RSVP-TE session.
+          type: integer
+          x-field-uid: 3
+        label_in:
+          description: >-
+            The label received by RSVP-TE ingress.
+          type: integer
+          x-field-uid: 4
+        label_out:
+          description: >-
+            The label assigned by RSVP-TE egress.
+          type: integer
+          x-field-uid: 5
+        session_status:
+          description: >-
+            Operational state of the RSVP LSP.
+          type: string
+          x-field-uid: 6
+          x-enum:
+            up:
+              x-field-uid: 1
+            down:
+              x-field-uid: 2
+        last_flap_reason:
+          description: >-
+            The reason for the last flap of this RSVP session.
+          type: string
+          x-field-uid: 7
+          x-enum:
+            resv_tear:
+              x-field-uid: 1
+            path_tear:
+              x-field-uid: 2
+            path_timeout:
+             x-field-uid: 3
+        up_time:
+          description: >-
+            The tunnel UP time in milli seconds.
+          type: integer
+          format: int64
+          x-field-uid: 8
+
+    RsvpLsp.Ipv4Rro:
+      description: >-
+        This contains the list of MPLS RRO objects associated with the traffic engineering tunnel.
+      type: object
+      properties:
+        address:
+          description: >-
+            The origin IPV4 address of RSVP session.
```

Comment:
Probably incorrect. Should probably be the IPv4 address of the interface or router through which the LSP has transited.